src/index.js

```diff
@@ -43,6 +43,18 @@ export default class AutoCheckElement extends HTMLElement {
     return input instanceof HTMLInputElement ? input : null
   }
 
+  get note(): HTMLElement {
+    let note = this.querySelector(this.noteSelector)
+    if (!note) {
+      note = document.createElement('p')
+    }
+    return note
+  }
+
+  get noteSelector(): string {
+    return this.getAttribute('data-note-selector') || 'p, .error, .warning'
```

Comment:
I don't think this should fallback to such generic selectors. Setting `innerHTML` should be very intentional. In `<details-menu>` we use `data-menu-button-text` for `textContent` and `data-menu-button-contents` for `innerHTML`.

Reply:
Yeah I was trying to be a bit cheeky so I didn't have to update all the instances in `github/github` 😄

Reply:
And it was a bit simpler then having a note and then an error/warning element. I'm gonna tackle it and see if I can make them separate but still have the API simple.